package/MDAnalysis/coordinates/null.py

```diff
@@ -0,0 +1,44 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""
+Null output --- :mod:`MDAnalysis.coordinates.null`
+==================================================
+
+The :class:`NullWriter` provides a Writer instance that behaves like
+any other writer but effectively ignores its input and does not write
+any output files. This is similar to writing to ``/dev/null``.
```

Comment:
You should add that this class exists to help writing generic analysis code that works with in-memory and normal trajectories.